src/openms/include/OpenMS/ANALYSIS/OPENSWATH/PeakIntegrator.h

```diff
@@ -119,6 +132,13 @@ namespace OpenMS
       PeakShapeMetrics_& peakShapeMetrics
     );
 
+    void calculatePeakShapeMetrics(
+      const MSSpectrum& spectrum,
+      const double& left,
+      const double& right,
+      PeakShapeMetrics_& peakShapeMetrics
```

Comment:
Does PeakShapeMetrics_ need to be passed in as a parameter since it is internal?  Removing this as a parameter will make the pyOpenMS wrapper much easier as well.

Reply:
Currently it's not a class' member, so it was necessary to pass it.
I will move it and then it won't be necessary to pass it, anymore.